controllers/projects_controller.go

```diff
@@ -280,3 +280,40 @@ func (u *Projects) FetchGithub(c web.C, w http.ResponseWriter, r *http.Request)
 		return
 	}
 }
+
+func (u *Projects) Webhook(c web.C, w http.ResponseWriter, r *http.Request) {
```

Comment:
__[golint]__
* [golint] exported method Projects.Webhook should have comment or be unexported